package.json

```diff
@@ -4,7 +4,7 @@
 	"description": "An extension to help generate the minimal code to be used while implementing clean architecture.",
 	"version": "1.0.0",
 	"engines": {
-		"vscode": "^1.62.0"
+		"vscode": "^1.61.0"
```

Comment:
@bssughosh 

I made this change

Reply:
Why was this required?

Reply:
My VS code version was 1.61

So tried it with this 
It worked after this 

Ill update my VSCode version and update this